app/views/reports/call_volumes.html.erb

```diff
@@ -18,6 +18,17 @@
 
       <div class="form-group">
         <%= f.submit 'Filter', class: 'btn btn-default t-filter-button' %>
+        <%=
+          link_to(
+            'Export CSV',
+            call_volumes_path(
+              start_date: @call_volumes.start_date,
```

Comment:
I wonder if we can wrap these parameters in `call_volumes:` so we can drop the `|| params[:start_date]` shenanigans?